### PR TITLE
Adding updated guidance for the translation process

### PR DESCRIPTION
**Pull request by juliaelman**
This pull request updates the translation guidance for secure.login.gov and login.gov. It helps to make translations easier and more accurate for everyone on the team to be able to make translations if they need to.

### CODE REVIEW

File: _articles/appdev-i18n.md [hunk 1]
```diff
@@ -43,9 +43,9 @@ As new screens are added or existing views are updated, the set of localized str
 
 ### When should strings be added?
 
-Page copy will often change through the process of design iteration and by feedback from others on the team. It is up to the developer's discretion when to add strings to `config/locales`, as opposed to using temporary text in an in-progress implementation.
+During the design process, the page copy may often change through iteration and feedback from the team and/or directly from the public. UX Team members will include layouts that display English, French and Spanish translations prior to handing off for Engineering to build. You can read more about that process here:  
 
-However, keep in mind:
+NOTE:
```
Comment (zachmargolis) [inline _articles/appdev-i18n.md:48]: As a personal style thing, I think bold communicates emphasis better than all-caps?

```suggestion
**Note**:
```

File: _articles/appdev-i18n.md [hunk 2]
```diff
@@ -61,12 +61,8 @@ You can also take advantage of some automated tooling to assist in this process:
...
 
-As mentioned earlier in this guide, the same YAML structure should exist for all supported locales, including the translated text values. Even if you are not a native speaker of the language, you should plan to include a translated value.
+Translations are requested when content is added or updated on secure.login.gov and login.gov. You can find specific guidance on how to request translations in the [Login.gov guidance document](https://docs.google.com/document/d/16RVO4Gr1bBTt8RV14Jsi1U77lKCBWRagowZbjdsXQ0w/edit).
 
-This can be done by…
-
-1. Reaching out to colleagues within TTS who are native speakers of the language
-2. Or, making a best effort at a translation using a service like [Google Translate](https://translate.google.com/)
-3. Or, using the available professional translation services with whom Login.gov contracts. Contact Amber Van Amburg on Slack for more information.
+You can find specific guidance on how to request translations in the [Login.gov guidance document](https://docs.google.com/document/d/16RVO4Gr1bBTt8RV14Jsi1U77lKCBWRagowZbjdsXQ0w/edit).
```
Comment (zachmargolis) [inline _articles/appdev-i18n.md:68]: this second para/sentence seems redundant? I'd just remove it, all the same info is right above it?

**Review by zachmargolis — APPROVED** [review]
LGTM! Some small nits, nothing major, thanks for updating this!

File: _articles/appdev-i18n.md (hunk 2 above)
Comment (juliaelman) [reply _articles/appdev-i18n.md]: Updated.

File: _articles/appdev-i18n.md [hunk 3]
```diff
@@ -1,12 +1,24 @@
 ---
-title: "Internationalization (i18n)"
+title: "Translation process"
 layout: article
 description: "Process and guidelines for localization and string translation"
```
Comment (zachmargolis) [inline _articles/appdev-i18n.md:4]: If we take "i18n" out of the title, can we add it to the description? The reason I like it there is it's helpful for searching and cmd-f on the homepage to find the right article
```suggestion
title: "Translation process"
layout: article
description: "Process and guidelines for localization and string translation (i18n)"
```

**Review by zachmargolis — APPROVED** [review]
LGTM!

**Pull request merged**